Update documentation about LZO and LZ4 modules (#85)
fox-it/dissect-docs: docs/source/projects/dissect.btrfs/index.rst

    $ pip install dissect.btrfs

> This project decompresses LZO compressed file systems and can use a faster, native (Rust-based) LZO implementation,
> instead of a slower pure Python implementation. Both implementations are provided by :doc:`/projects/dissect.util/index`.
> Pre-build wheels are available for most common platforms and the native implementation will automatically be used if available.
> In the rare case that a pre-build wheel is not available, please refer to :doc:`/projects/dissect.util/index` for build instructions.

Reviewer: Now this reads as if the whole file system is decompressed. Maybe something along the lines of:
```suggestion
This project decompresses LZO compressed file streams. For this it uses a native rust-based LZO implementation, instead of a slower pure Python implementation. Both implementations are provided by :doc:`/projects/dissect.util/index`.
Pre-build wheels are available for most common platforms and the native implementation will automatically be used if available.
In the rare case that a pre-build wheel is not available, please refer to :doc:`/projects/dissect.util/index` for build instructions.
```

Author: ```suggestion
This project supports file systems with LZO compression. It can use either a fast Rust-based LZO implementation, or a slow pure Python implementation. Both implementations are provided by :doc:`/projects/dissect.util/index`, and the faster implementation will automatically be used if available.
Pre-build wheels are available for most common platforms. In the rare case that a pre-build wheel is not available, please refer to :doc:`/projects/dissect.util/index` for build instructions.
```
